tools/isobuild/resolver.ts

```diff
@@ -21,18 +21,25 @@ import {
   optimisticReadJsonOrNull,
 } from "../fs/optimistic";
 
-const nativeModulesMap: Record<string, string> = Object.create(null);
-const nativeNames = Object.keys((process as any).binding("natives"));
-
-// Node 0.10 does not include process as a built-in module, but later
-// versions of Node do, and we provide a stub for it on the client.
-nativeNames.push("process");
+// Note: the source code for the following function was copied from the 'builtin-modules' npm package (ver. 3.1.0)
+function getBuiltinModules() {
+  'use strict';
+  const {builtinModules} = require('module');
+
+  const blacklist = [
+    'sys'
+  ];
+
+  // eslint-disable-next-line node/no-deprecated-api
+  return (builtinModules || Object.keys((process as any).binding('natives')))
```

Comment:
I think we can leave out the fallback to process.bind('natives') here, since we know the node version. It's deprecated anyway ( https://nodejs.org/dist/latest-v12.x/docs/api/deprecations.html#deprecations_dep0111_process_binding )

Reply:
OK. The code has been changed to reflect this simplification.